src/steamship/plugin/blockifier.py

```diff
@@ -22,8 +22,8 @@ class Blockifier(PluginService[RawDataPluginInput, BlockAndTagPluginOutput], ABC
     # noinspection PyUnusedLocal
     def __init__(self, client: Client = None, config: Dict[str, Any] = None):
         super().__init__(client, config)
-        if config:
-            self.config = self.config_cls()(**config)
+        if self.config:
```

Comment:
I'm pretty confident the prior version is actually correct? I just followed up the `super()` chain and nothing above this class actually does anything with the `config` passed in. It's here that we have to call `self.config_cls()(**config)` if it's non None